.openshift-ci_cd/Jenkinsfile

```diff
@@ -32,7 +32,7 @@ node ('redhatdistortion-nodejs') {
 
     stage 'Run Integration Tests'
     sh 'npm config set prefix=$HOME/node && export PATH=$HOME/node/bin:$PATH && npm install -r package.json'
-    sh 'IP_=`oc get endpoints | grep nodejs-ex-dev | awk \'{print $2}\'| cut -d\':\' -f1` && IP=$IP_ PORT=8080 node_modules/mocha/bin/mocha --timeout 5000 tests/app_test.js --reporter xunit | tee report.xml'
+    sh 'ROUTE=`oc get routes | grep nodejs-ex-dev | awk \'{print $2}\'` && IP=$ROUTE node_modules/mocha/bin/mocha --timeout 5000 tests/app_test.js --reporter xunit | tee report.xml'
```

Comment:
Would something like this work?
```bash
IP=http://nodejs-ex-dev:80/ node_modules/mocha/bin/mocha --timeout 5000 tests/app_test.js --reporter xunit > report.xml
```

Because you are actually using routes not the internal DNS (https://docs.openshift.org/latest/architecture/additional_concepts/networking.html#openshift-origin-dns) I have mentioned.